sstables/mc/parquet_writer.hh

```diff
@@ -0,0 +1,973 @@
+#pragma once
+
+#include <parquet4seastar/file_writer.hh>
+#include <parquet4seastar/y_combinator.hh>
+#include <parquet4seastar/overloaded.hh>
+#include <schema.hh>
+#include <keys.hh>
+#include <mutation_fragment.hh>
+#include "sstables/mc/writer.hh"
+
+extern logging::logger parquet_logger;
```

Comment:
style note: having to declare an extern logger is a symptom of having too much implementation inside a header. Functions with complicated logic should be moved to a .cc file to remove bloat - and functions which have to log their behavior are by definition complicated enough.